### PR TITLE
Infrared emitter lasers no longer float off into space. Fixes #25317

### LINKED ISSUE #25317: Lasers from infrared emitters float off into space
### Exploit Reports

- [x] I confirm this issue is not an exploit. (Required)

### BYOND Version

515.1636

### Issue Description

Activating a infrared emitter with the visible laser enabled and pointing it at a large 0g environment will cause the laser beam to float away from the source 

### What did you expect to happen?

For the laser to stay fixed

### What happened instead?

The laser slowly flew away from me

### Why is this bad/What are the consequences?

It's quite silly, and likely also stops the laser from triggering in its intended area. 

### Steps to reproduce the issue.

Put infrared emitter on visible mode
Point laser out exterior window
Watch the laser float away into deep spess

### When did the problem start happening?

No idea. 

### Extra information

_No response_

### Relevant log output/runtime error

_No response_

### PR DESCRIPTION
**Pull request by Daylight2**
## What Does This PR Do
Fixes #25317.
Also makes atoms_movable.dm slightly more readable.

## Why It's Good For The Game
Bug bad.

## Testing
Pointed a laser out a window

## Changelog
:cl:
fix: Fixed infrared emitter lasers floating off into space.
/:cl:

### CODE REVIEW

File: code/game/atoms_movable.dm [hunk 1]
```diff
@@ -355,6 +355,9 @@
 	if(has_gravity(src))
 		return 1
 
+	if(anchored)
+		return 1
+
 	if(pulledby && !pulledby.pulling)
 		return 1
```
Comment (DGamerL) [inline code/game/atoms_movable.dm:362]:
```suggestion
		return TRUE

	if(anchored)
		return TRUE

	if(pulledby && !pulledby.pulling)
		return TRUE
```

**Review by DGamerL — COMMENTED** [review]
Please also change the `1` to `TRUE` and `0` to `FALSE` on lines 365 to 370

File: code/game/atoms_movable.dm (hunk 1 above)
Comment (Daylight2) [reply code/game/atoms_movable.dm]: No longer touching that file, but sure

File: code/game/atoms_movable.dm [hunk 2]
```diff
@@ -199,7 +199,7 @@
 	Moved(old_loc, direction, TRUE)
 
 /atom/movable/Move(atom/newloc, direct = 0, movetime)
-	if(!loc || !newloc) return 0
+	if(!loc || !newloc) return FALSE
```
Comment (DGamerL) [inline code/game/atoms_movable.dm:202]:
```suggestion
	if(!loc || !newloc) 
		return FALSE
```

**Review by DGamerL — APPROVED** [review]

**Pull request merged**